resources/warnings.json

```diff
@@ -11277,7 +11277,7 @@
     "versions": [
       {
         "lastVersion": "1.5",
-        "pattern": ".*"
+        "pattern": "1[.][0-5](|[.-].*)"
```

Comment:
Release 244 does not have the fix but would be considered unaffected by this pattern. Please also update the `lastVersion`.

Reply:
Fixed in 546a872514aeb2a76b4f2fac78244d63cf11c949 .  Thanks for detecting that mistake!

Reply:
@daniel-beck I believe this is ready for another review.  Thanks again for detecting my mistake in the `lastVersion` and the matching pattern.